docs/spec/operators/localmanager.md

```diff
@@ -0,0 +1,107 @@
+---
+title: LocalManager
+---
+
+The LocalManager operator is used by the `ig` binary. This operator keeps track
+of the containers on the system talking to the different container runtimes and
+using the fanotify mechanism to detect when containers are created. This
+information is used to:
+- Update the eBPF maps used to filter events in eBPF by container name.
+- Enrich events with container information (see below).
+- Attach networking programs to specific containers according to the filtering
+  options passed by the user.
+
+This operator uses the mount or network namespace inode IDs to enrich events
+with the following fields:
+
+- runtime
+  - runtime name
+  - container name
+  - container ID
+  - container image name
+  - container image digest
+  - container started time
+
+It also adds some Kubernetes information gathered from the container runtimes,
+but it doesn't talk to the kube-apiserver unless the
+[`enrich-with-k8s-apiserver`](#enrich-with-k8s-apiserver) parameter is set.
+- k8s
+  - container name
+  - pod name
+  - namespace
+  - pod labels
+  - owner (only when using `enrich-with-k8s-apiserver`)
+
+## Priority
+
+-1
+
+## Global Parameters
```

Comment:
Why global params don't have the fully qualified name? That's what how we are stating params can be passed through [environment variables and config file](https://www.inspektor-gadget.io/docs/latest/reference/configuration):

> You can use environment variables to configure the behavior of the run command. The environment variables use fully qualified names (as in the [configuration file](https://www.inspektor-gadget.io/docs/latest/reference/configuration#configuration-file)) with the prefix `INSPEKTOR_GADGET_`.
>

However, I think I found an inconsistency. Let's use runtimes param as example:

- Env variable: `INSPEKTOR_GADGET_OPERATOR_LOCALMANAGER_RUNTIMES`
- Config file: `operator.localmanager.runtimes`
- But if it were an instance param: `operator**s**.localmanager.runtimes`

cc @flyth